libs/accounts/rate-limit/src/lib/rate-limit.provider.ts

```diff
@@ -0,0 +1,56 @@
+/* This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this
+ * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+import { ConfigService } from '@nestjs/config';
+import { parseConfigRules, RateLimitConfig } from './config';
+import { RateLimit } from './rate-limit';
+import Redis from 'ioredis';
+import { StatsD, StatsDService } from '@fxa/shared/metrics/statsd';
+import { Provider } from '@nestjs/common';
+
+export const RateLimitRedisClient = Symbol('RATE_LIMIT__REDIS_CLIENT');
+export const RateLimitRedisProvider: Provider = {
+  provide: RateLimitRedisClient,
+  useFactory: (config: ConfigService) => {
+    const redisConfig = config.get<any>('redis');
```

Comment:
Any reason we can't specify the type of the config value?

Reply:
Good call. Let me fix that.